src/libraries/System.Runtime/tests/System/ArgumentOutOfRangeExceptionTests.cs

```diff
@@ -65,5 +66,70 @@ public static void Ctor_String_Object_String()
             Assert.Contains(argumentName, exception.Message);
             Assert.Contains(argumentValue.ToString(), exception.Message);
         }
+
+        private static Action ZeroHelper<T>(T value) where T : INumberBase<T> => () => ArgumentOutOfRangeException.ThrowIfZero(value);
+        private static Action NegativeOrZeroHelper<T>(T value) where T : INumberBase<T> => () => ArgumentOutOfRangeException.ThrowIfNegativeOrZero(value);
+        private static Action GreaterThanHelper<T>(T value, T other) where T : IComparable<T> => () => ArgumentOutOfRangeException.ThrowIfGreaterThan(value, other);
+        private static Action GreaterThanOrEqualHelper<T>(T value, T other) where T : IComparable<T> => () => ArgumentOutOfRangeException.ThrowIfGreaterThanOrEqual(value, other);
+        private static Action LessThanHelper<T>(T value, T other) where T : IComparable<T> => () => ArgumentOutOfRangeException.ThrowIfLessThan(value, other);
+        private static Action LessThanOrEqualHelper<T>(T value, T other) where T : IComparable<T> => () => ArgumentOutOfRangeException.ThrowIfLessThanOrEqual(value, other);
+
+        [Fact]
+        public static void GenericHelpers_ThrowIfZero_Throws()
+        {
+            Assert.Throws<ArgumentOutOfRangeException>(ZeroHelper<int>(0));
```

Comment:
These should all use AssertExtensions instead of Assert and validate the parameter name.